awslimits/support.py

```diff
@@ -100,30 +105,27 @@ def load_tickets():
 
 def get_limit_types():
     limit_types = []
-    checker = AwsLimitChecker(region=settings.REGION_NAME, account_role=settings.ROLE_ARN)
-    for service, service_limits in checker.get_limits(use_ta=False).items():
+    checker = AwsLimitChecker(region=settings.REGION_NAME, account_id=settings.ACCOUNT_ID, account_role=settings.ACCOUNT_ROLE)
```

Comment:
I think it's a good idea to extract this variable to a method and reuse in the other functions (like you did for `get_<xyz>_table`).